content/millennium/r4/proprietary-codes.md

```diff
@@ -78,6 +78,13 @@ The Appointment Resource supports proprietary codes for:
 
 * Appointment.serviceType codes are maintained in [Code Set 14249 Scheduling Appointment Type Synonyms](#code-set-14249-scheduling-appointment-type-synonyms)
 
+<h3>RelatedPerson</h3>
+
+The RelatedPerson Resource supports proprietary codes for:
+
+* RelatedPerson.communication.language codes are maintained in [Code Set 36 Languages](#code-set-36-languages)
+* RelatedPerson.relationship codes are maintained in [Code Set 351 Person Relationship Types](#code-set-351-person-relationship-types) and [Code Set 40 Person     Relationhip Types](#code-set-40-person-relationship)
```

Comment:
remove space in link to cs 40. i would like to order the links numerically

Reply:
removed the space https://github.com/cerner/fhir.cerner.com/pull/254/commits/a6cc265db68cabf4cb1d9abc1dec64f71d1c0b86#diff-f1a7812a3d6e9f4049fb6005eecfb1f5R78

Reply:
ordered numerically https://github.com/cerner/fhir.cerner.com/pull/254/commits/53300d0b6211f4b0c77925c1e5ab9b78e4010aff#diff-f1a7812a3d6e9f4049fb6005eecfb1f5R78

Reply:
change Relationhip to Relationship

Reply:
corrected spelling
https://github.com/cerner/fhir.cerner.com/pull/254/commits/a61676ce8e25adc46975ea912b5016edde8a8b6e#diff-f1a7812a3d6e9f4049fb6005eecfb1f5R78